modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/OnboardingPlusFeaturesPage.kt

```diff
@@ -0,0 +1,465 @@
+                modifier = Modifier
+                    .height(48.dp)
+                    .width(48.dp)
+            )
+
+            Spacer(Modifier.height(12.dp))
+
+            Column {
+
+                IconRow(Modifier.padding(horizontal = 24.dp))
+
+                Spacer(Modifier.height(36.dp))
+
+                TextH10(
+                    text = stringResource(LR.string.onboarding_upgrade_everything_you_love_about_pocket_casts_plus),
+                    color = Color.White,
+                    modifier = Modifier.padding(horizontal = 24.dp),
+                )
+
+                Spacer(Modifier.height(12.dp))
+
+                TextP40(
+                    text = stringResource(LR.string.onboarding_upgrade_exclusive_features_and_options),
+                    color = Color.White.copy(alpha = 0.8f),
+                    modifier = Modifier.padding(horizontal = 24.dp),
+                )
+
+                Spacer(Modifier.height(58.dp))
+
+                FeatureRow(scrollAutomatically = state.scrollAutomatically)
+
+                Spacer(Modifier.weight(1f))
+                Spacer(Modifier.height(36.dp))
+
+                PlusRowButton(
+                    text = stringResource(LR.string.onboarding_upgrade_unlock_all_features),
+                    onClick = onUpgradePressed,
+                    modifier = Modifier.padding(horizontal = 24.dp),
+                )
+
+                Spacer(Modifier.height(16.dp))
+
+                PlusOutlinedRowButton(
+                    text = stringResource(LR.string.not_now),
+                    onClick = onNotNowPressed,
+                    modifier = Modifier.padding(horizontal = 24.dp),
+                )
+
+                Spacer(Modifier.height(16.dp))
+            }
+        }
+    }
+}
+
+@Composable
+private fun IconRow(modifier: Modifier = Modifier) {
+    Row(modifier) {
+        Icon(
+            painter = painterResource(R.drawable.pocket_casts_white),
+            contentDescription = null,
+            tint = Color.White,
+        )
+
+        Spacer(Modifier.width(8.dp))
+
+        Icon(
+            painter = painterResource(R.drawable.plus_bw),
+            contentDescription = null,
+            tint = Color.White,
+        )
+    }
+}
+
+@Composable
+private fun FeatureRow(scrollAutomatically: Boolean) {
+
+    // Not using rememberLazyListState() because we want to reset
+    // the scroll state on orientation changes so that the hardcoded column
+    // is redisplayed, which insures the height is correctly calculated. For that
+    // reason, we want to use remember, not rememberSaveable.
+    val state = remember { LazyListState() }
+
+    val localConfiguration = LocalConfiguration.current
+    LaunchedEffect(scrollAutomatically) {
+        if (scrollAutomatically) {
+            // This seems to get a good scroll speed across multiple devices
+            val scrollDelay = max(1L, (1000L - localConfiguration.densityDpi) / 125)
```

Comment:
I'm using this math because just using a constant number made the row scroll to quickly on some devices and too slowly on others. This math seemed to give a reasonable result across devices, but I'm sure it could be improved, so let me know if you have any suggestions.